examples/remarketing/upload_conversion_adjustment.py

```diff
@@ -0,0 +1,124 @@
+        conversion_adjustment_type_enum.RESTATEMENT):
+        conversion_adjustment.restatement_value.adjusted_value.value = (
+            float(restatement_value))
+        conversion_adjustment.restatement_value.currency_code.value = 'USD'
+
+    # Set the Gclid Date
+    conversion_adjustment.gclid_date_time_pair.gclid.value = gcl_id
+    conversion_adjustment.gclid_date_time_pair.conversion_date_time.value = (
+        conversion_time)
+
+    # Try the upload
+    conversion_adjustment_upload_service = (
+        client.get_service('ConversionAdjustmentUploadService', version='v2')
+    )
+
+    try:
+        response = (
+            conversion_adjustment_upload_service.
+            upload_conversion_adjustments(customer_id,
+                                          [conversion_adjustment],
+                                          partial_failure=True)
+        )
+        conversion_adjustment_result = response.results[0]
+        print(f'Uploaded conversion that occurred at '
+              f'"{conversion_adjustment_result.adjustment_date_time.value}" '
+              f'from Gclid '
+              f'"{conversion_adjustment_result.gclid_date_time_pair.gclid.value}"'
+              f' to "{conversion_adjustment_result.conversion_action.value}"')
+
+    except GoogleAdsException as ex:
+        print(f'Request with ID "{ex.request_id}" failed with status '
+              f'"{ex.error.code().name}" and includes the following errors:')
+        for error in ex.failure.errors:
+            print(f'\tError with message "{error.message}".')
+            if error.location:
+                for field_path_element in error.location.field_path_elements:
+                    print(f'\t\tOn field: {field_path_element.field_name}')
+        sys.exit(1)
+
+
+if __name__ == '__main__':
+    # GoogleAdsClient will read the google-ads.yaml configuration file in the
+    # home directory if none is specified.
+    google_ads_client = GoogleAdsClient.load_from_storage()
+
+    parser = argparse.ArgumentParser(
+        description='Upload an offline conversion.')
+    # The following argument(s) should be provided to run the example.
+    parser.add_argument('-c', '--customer_id', type=str,
+                        required=True, help='The Google Ads customer ID.')
+    parser.add_argument('-a', '--conversion_action_id', type=str,
+                        required=True, help='The conversion action ID.')
+    parser.add_argument('-g', '--gcl_id', type=str,
+                        required=True, help='The Google Click Identifier ID.')
+    parser.add_argument('-d', '--adjustment_type', type=str,
+                        required=True, help='The Adjustment type.')
+    parser.add_argument('-t', '--conversion_time', type=str,
+                        required=True, help='The conversion time.')
+    parser.add_argument('-v', '--adjustment_time', type=str,
+                        required=True, help='The adjustment time.')
```

Comment:
The helpers miss information compare to the golden (e.g. the format and constraint of "conversion_time")

Reply:
Looks like it has been fixed